dbms/src/Flash/Mpp/ExchangeReceiver.h

```diff
@@ -189,8 +193,10 @@ class ExchangeReceiverBase
     std::mutex mu;
     std::condition_variable cv;
     /// should lock `mu` when visit these members
-    Int32 live_connections;
     Int32 live_local_connections;
+    Int32 actual_created_connections;
```

Comment:
Consider use original way that initialize live_connections as source num? So many status makes it a little complex.

Reply:
It's ok to be considered later, fix ut asap.

Reply:
+1